pymbolic/primitives.py

```diff
@@ -1073,63 +1104,69 @@ class FunctionSymbol(AlgebraicLeaf):
 class Call(AlgebraicLeaf):
     """A function invocation.
 
-    .. attribute:: function
-
-        A :class:`Expression` that evaluates to a function.
-
-    .. attribute:: parameters
-
-        A :class:`tuple` of positional parameters, each element
-        of which is a :class:`Expression` or a constant.
-
+    .. autoattribute:: function
+    .. autoattribute:: parameters
     """
     function: ExpressionT
+    """A :class:`Expression` that evaluates to a function."""
+
     parameters: tuple[ExpressionT, ...]
+    """
+    A :class:`tuple` of positional parameters, each element
+    of which is a :class:`Expression` or a constant.
+    """
 
 
 @expr_dataclass()
 class CallWithKwargs(AlgebraicLeaf):
     """A function invocation with keyword arguments.
 
-    .. attribute:: function
-
-        A :class:`Expression` that evaluates to a function.
-
-    .. attribute:: parameters
-
-        A :class:`tuple` of positional parameters, each element
-        of which is a :class:`Expression` or a constant.
-
-    .. attribute:: kw_parameters
-
-        A dictionary mapping names to arguments, , each
-        of which is a :class:`Expression` or a constant,
-        or an equivalent value accepted by the :class:`dict`
-        constructor.
+    .. autoattribute:: function
+    .. autoattribute:: parameters
+    .. autoattribute:: kw_parameters
     """
 
     function: ExpressionT
+    """An :class:`Expression` that evaluates to a function."""
+
     parameters: tuple[ExpressionT, ...]
+    """A :class:`tuple` of positional parameters, each element
+    of which is a :class:`Expression` or a constant.
+    """
+
     kw_parameters: Mapping[str, ExpressionT]
+    """A dictionary mapping names to arguments, , each
+    of which is a :class:`Expression` or a constant,
+    or an equivalent value accepted by the :class:`dict`
+    constructor."""
```

Comment:
```suggestion
    of which is a :class:`Expression` or a constant.
    """
```
? Since neither the type annotation nor the `__post_init__` transforms it into a dict.

Reply:
Done.